Annotato/Annotato/Auth/FirebaseAuth.swift

```diff
@@ -0,0 +1,78 @@
+import Firebase
+import AnnotatoSharedLibrary
+
+class FirebaseAuth: AnnotatoAuthService {
+    static var isAlreadyConfigured = false
```

Comment:
```suggestion
    private static var isAlreadyConfigured = false
```

Does this have to be non-private?